traffic_ops/traffic_ops_golang/ats/config.go

```diff
@@ -1,5 +1,15 @@
 package ats
 
+import (
+	"database/sql"
+	"fmt"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
+	"github.com/influxdata/platform/kit/errors"
```

Comment:
I think this is supposed to simply be "errors". My trafficops seems to run fine with it though.

Reply:
Ah, I missed this. Yeah, this should be changed to the Go standard library.

In general, we've been trying to use standard errors, and not wrap them in objects, and using unique human-readable strings, wrapped with context; and returning separate enums for anything that needs switched on by the code. See https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully

We'd need to have a discussion as a project, to move away from that. Personally, I've been writing Go for some 10 years now, and I lean heavily toward what Dave Cheney recommends. All the other options are far more difficult to work with.

It looks like that library is no longer maintained, anyway.